bundles/model/org.eclipse.smarthome.model.item/src/org/eclipse/smarthome/model/item/internal/GenericItemProvider.java

```diff
@@ -340,8 +337,6 @@ private void internalDispatchBindings(BindingConfigReader reader, String modelNa
     }
 
     /**
-     * {@inheritDoc}
-     * <p>
```

Comment:
I am not sure about this one.
Wasn't the previous approach to use the parent JavaDoc and additionally an sentence?
Isn't now the parent JavaDoc overwritten by the sentence?

I can't check it ATM but it feels like "parent + custom" before and "custom" now. WDYT?